backend/tests/test_user_api.py

```diff
@@ -0,0 +1,148 @@
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+
+from rest_framework.test import APIClient
+from rest_framework import status
+
+CREATE_USER_URL = reverse('meal:create_user')
+TOKEN_URL = reverse('meal:login')
+ME_URL = reverse('meal:me')
+
+
+def create_user(**kwargs):
+    return get_user_model().objects.create_user(**kwargs)
+
+
+class PublicUserApiTests(TestCase):
+    """ Test the users API (public)"""
+
+    def setUp(self):
+        self.client = APIClient()
+
+    def test_create_valid_user_success(self):
+        """Test creating user with valid payload is successful"""
+        payload = {
+            'username': 'sebix',
```

Comment:
Lets create test tables for this test suite, this will simplify this file a lot and will help with future tests.